scholly/base/src/components/publications/Publications.js

```diff
@@ -1,69 +1,70 @@
 /**
  * The Publications component displays a list of publications.
  */
-import React, { Fragment, useState } from 'react';
+import React, { Fragment } from 'react';
 import { Accordion } from 'react-bootstrap';
 import LayoutAllPublications from './publicationsLayout/LayoutAllPublications';
 import LayoutByCategory from './publicationsLayout/LayoutByCategory'
 import { TEAM_PUBLICATIONS } from '../../global/data';
-import PublicationsDropdown from './publicationsLayout/PublicationsDropdown';
 import { WEB_PAGES } from '../../global/data';
-import { layoutOption, sortingOption, defaultOption } from '../../config/publications';
+import { layoutOptions, sortingOptions, defaultOption } from '../../config/publications';
 
 const Publications = () => {
-  const { publicationOptions } = WEB_PAGES
-  const [ options, setOptions ] = useState(publicationOptions ? publicationOptions : defaultOption);
+  const options = WEB_PAGES.publicationOptions ?? defaultOption
   const sortPublications = (teamPublications, option) => {
     switch (option) {
-        case sortingOption.AUTHOR:
-            teamPublications.sort((a, b) =>
-                a.authors[0].toLowerCase() > b.authors[0].toLowerCase() ? 1 : -1
-            );
-            break;
-        case sortingOption.TITLE:
-            // publication title
-            teamPublications.sort((a, b) =>
-                a.title.toLowerCase() > b.title.toLowerCase() ? 1 : -1
-            );
-            break;
-        case 'Category Title':
-            // journal or conference title
-            teamPublications.sort((a, b) =>
-                a.category.categoryTitle.toLowerCase() >
-                b.category.categoryTitle.toLowerCase()
-                ? 1
-                : -1
-            );
-            break;
-        default:
-            // sort by title then year for consistency with the db
-            teamPublications.sort((a, b) =>
-                a.title.toLowerCase() > b.title.toLowerCase() ? 1 : -1
-            );
-            teamPublications.sort((a, b) => (a.year > b.year ? -1 : 1));
-            break;
-        }
-        return teamPublications
+      case sortingOptions.AUTHOR:
```

Comment:
if we're removing sorting in the `base`, shouldn't this functionality be removed too?

Reply:
As the publications in the db are't sorted, this function is still needed for sorting the publication in the first rendering based on the layout and sorting preference stored in the db

Reply:
SGTM, relsving.